src/main.cpp

```diff
@@ -0,0 +1,48 @@
+#include "solve_quadratic.h"
+#include "test_quadratic.h"
+#include <stdio.h>
+#include "cstring"
+
+const char* TEST_ARG = "--test";
+//---------------------------------------------------------------------------//
+
+void AnalyzeStartupArguments (int argc, char *argv[]){
+    int argcCopy = argc;
+    bool flagTestSolver = 0;
```

Comment:
Don't ever call flag variables as flags. Usually you can start with is (but in this case something like useTesting)